src/services/codefixes/inferFromUsage.ts

```diff
@@ -150,35 +155,67 @@ namespace ts.codefix {
             return;
         }
 
-        const types = inferTypeForParametersFromUsage(containingFunction, sourceFile, program, cancellationToken) ||
-            containingFunction.parameters.map(p => isIdentifier(p.name) ? inferTypeForVariableFromUsage(p.name, program, cancellationToken) : undefined);
+        const symbols = inferTypeForParametersFromUsage(containingFunction, sourceFile, program, cancellationToken) ||
+            containingFunction.parameters.map(p => ({
+                declaration: p,
+                type: isIdentifier(p.name) ? inferTypeForVariableFromUsage(p.name, program, cancellationToken) : undefined
+            } as ParameterInference));

```

Comment:
`as` is unsafe since it allows you to omit or misspell properties. Better to add a type argument to `map` or a return type annotation to the arrow function.